data/data/vsphere/master/main.tf

```diff
@@ -1,31 +1,47 @@
 locals {
+  // limitation of baremetal-runtimecfg.  The hostname must be master
+  master_name = "master"
   description = "Created By OpenShift Installer"
 }
 
-resource "vsphere_virtual_machine" "vm" {
-  count = var.instance_count
+provider "vsphere" {
+  user                 = var.vsphere_username
+  password             = var.vsphere_password
+  vsphere_server       = var.vsphere_url
+  allow_unverified_ssl = false
+}
+
+provider "vsphereprivate" {
```

Comment:
If the `vsphereprivate_import_ova` resource is not being used then this isn't required
https://github.com/openshift/installer/blob/master/data/data/vsphere/main.tf#L44-L53